pkg/knuu/knuu_test.go

```diff
@@ -147,3 +161,51 @@ func TestNew(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateOptions(t *testing.T) {
+	tests := []struct {
+		name    string
+		options Options
+		wantErr bool
+	}{
+		{
+			name: "MinioClient set without K8sClient",
+			options: Options{
+				MinioClient: &minio.Minio{},
+			},
+			wantErr: true,
+		},
+		{
+			name: "Both MinioClient and K8sClient set",
+			options: Options{
+				MinioClient: &minio.Minio{},
+				K8sClient:   &mockK8s{},
+			},
+			wantErr: false,
+		},
+		{
+			name: "TestScope and K8sClient not set",
+			options: Options{
+				TestScope: "",
+				K8sClient: nil,
+			},
+			wantErr: true,
```

Comment:
this can be removed as this is just the base case that should be valid.

Reply:
the test case should be validating that when they are both set but don't match, it errors.